src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java

```diff
@@ -115,5 +118,25 @@ public static void addCustomFieldsToRequest( CertificateRequest request, Certifi
 			});
 		}
 	}
+	
+	public static String getIpAddress() throws UnknownHostException {
+		
+		return  Inet4Address.getLocalHost().getHostAddress();
+		
+	}
+
+	public static void addApliClientInformation( com.venafi.vcert.sdk.connectors.cloud.CloudConnector.CertificateRequestsPayload payload ) {
```

Comment:
minor typo here. Method name should be addApiClientInformation()

Reply:
fixed on the latest commit.